### PR TITLE
Estimate Your Benefits Bottom Sheet Feature Flag 

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a developer, I need to create a toggle for the Estimate Your Benefits Bottom Sheet, so that it can be enabled / disabled without requiring a deployment.
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9403

## Testing done
local testing complete at `http://localhost:3000/flipper/features/gibct_eyb_bottom_sheet`

## Acceptance criteria
- [x] A Feature Flag is available to be used for the Estimate Your Benefits development.
- [x] The Feature Flag name is: gibctEybBottomSheet
- [x] The Feature Flag description is: "Panel that displays while the user is modifying inputs to give context to their currently estimated benefits until they reach the full your estimated benefits panel."
- [x] The Feature Flag is defaulted to the OFF state.
- [x] The bottom sheet is not displayed when the feature flag is in the off state.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

File: src/applications/gi/tests/data/feature-toggles.json [hunk 1]
```diff
@@ -61,6 +61,10 @@
       {
         "name": "gibctEstimateYourBenefits",
         "value": true
+      },
+      {
+        "name": "gibctEybBottomSheet",
+        "value": false
```
Comment (bkjohnson) [inline src/applications/gi/tests/data/feature-toggles.json:67]: The flag looks like it is set to false for the tests...

File: src/platform/testing/local-dev-mock-api/common.js [hunk 2]
```diff
@@ -70,7 +70,8 @@ const responses = {
         { name: 'ssoeEbenefitsLinks', value: false },
         { name: 'eduBenefitsStemScholarship', value: true },
         { name: 'edu_section_103', value: true },
-        { name: 'gibctEstimateYourBenefits', value: true },
+        { name: 'gibctEstimateYourBenefits', value: false },
+        { name: 'gibctEybBottomSheet', value: true },
```
Comment (bkjohnson) [inline src/platform/testing/local-dev-mock-api/common.js:74]: But the mock API seems to be returning true.

**Review by bkjohnson — APPROVED** [review]
The Acceptance Criteria states:

 > The Feature Flag is defaulted to the OFF state.

Some of the testing changes don't seem consistent with that, but maybe that isn't a concern.

File: src/applications/gi/tests/data/feature-toggles.json (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/tests/data/feature-toggles.json:67]: fixed it

File: src/applications/disability-benefits/686c-674/tests/e2e/686-test-data.json [hunk 3]
```diff
@@ -3,11 +3,7 @@
     "optionsSelection": {
       "addChild": true,
       "addSpouse": true,
-      "reportDivorce": true,
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/tests/e2e/686-test-data.json:6]: Does master need to be merged into your branch? This reverts work that was merged in this pull request - https://github.com/department-of-veterans-affairs/vets-website/pull/12854/files

File: src/applications/disability-benefits/686c-674/tests/e2e/test-helpers.js [hunk 4]
```diff
@@ -487,133 +487,6 @@ export const fillVeteranMarriageHistoryDetails = (client, data) => {
     );
 };
 
-export const fillDeceasedName = (client, data) => {
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/tests/e2e/test-helpers.js:490]: Same here. This reverts work merged in here - https://github.com/department-of-veterans-affairs/vets-website/pull/12854/files

File: src/applications/disability-benefits/bdd/config/form.js [hunk 5]
```diff
@@ -104,12 +104,12 @@ const formConfig = {
     environment.API_URL
   }/v0/disability_compensation_form/submit_all_claim`,
   submit: submitFormFor('disability-526EZ'),
-  trackingPrefix: 'disability-526EZ-bdd-',
```
Comment (micahchiang) [inline src/applications/disability-benefits/bdd/config/form.js:107]: @ndsprinkle - correct me if I'm wrong, but I believe this reverts work you recently merged in here - https://github.com/department-of-veterans-affairs/vets-website/pull/12810

File: src/applications/disability-benefits/bdd/form-entry.jsx [hunk 6]
```diff
@@ -6,7 +6,7 @@ import startApp from 'platform/startup';
 import routes from './routes';
 import reducer from './reducers';
 import manifest from './manifest.json';
-import analyticsEvents from './analytics-events';
```
Comment (micahchiang) [inline src/applications/disability-benefits/bdd/form-entry.jsx:9]: See above comment on PR #12810

File: src/applications/disability-benefits/wizard/createWizard.jsx [hunk 7]
```diff
@@ -3,10 +3,7 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { VA_FORM_IDS } from 'platform/forms/constants';
 
-const disabilityForms = new Set([
```
Comment (micahchiang) [inline src/applications/disability-benefits/wizard/createWizard.jsx:6]: See above comment on pr #12810

**Review by micahchiang — CHANGES_REQUESTED** [review]
I think this branch is out of date, since it reverts work that was recently merged in at least three separate pull requests. #12810, #12854, and #12834.

File: src/applications/disability-benefits/686c-674/tests/e2e/686-test-data.json (hunk 3 above)
Comment (ksantiagomorgan6) [reply src/applications/disability-benefits/686c-674/tests/e2e/686-test-data.json:6]: no it was a bad commit fixing it now

**Pull request closed without merging**